example/Python/src/ReflexGame/ReflexGame.lf

```diff
@@ -0,0 +1,192 @@
+target Python {
+    keepalive: true,
+    files: [gui.py]
+}
+
+preamble {=
```

Comment:
I think it's better to move this preamble to `RandomSource`. This makes the reactor a lot more self-contained.

Reply:
sounds good, I'll update that momentarily